package.json

```diff
@@ -27,10 +27,11 @@
   "scripts": {
     "build": "tsc",
     "postbuild": "cpy README.md ./dist && cpy package.json ./dist && cpy LICENSE ./dist",
-    "lint": "eslint . --max-warnings 0 --ext .js,.ts",
-    "lint:fix": "npm run lint -- --fix",
     "format": "prettier --write README.md \"{lib,docs,tests}/**/*.{js,ts,md}\"",
     "format:check": "prettier --check README.md \"{lib,docs,tests}/**/*.{js,json,yml,ts,md}\"",
+    "generate:configs": "ts-node tools/generate-configs",
```

Comment:
We need to generate this at some point in the CI and CD. I guess it makes sense to automatically run in a new `generate_docs` step within `pipeline` jobs, before the `release` one. Inside this job, we only run this `generate:configs` for now, but we can include more steps when we have more doc stuff automated. What do you think?

Reply:
FWIW, I added this to the commit hook in the NgRx ESLint plugin.
Just to keep the configs up-to-date in the repository.

Reply:
That's a good point, but then this is gonna be really easy to forget? I'm not sure where to include it, to be honest.

Reply:
I've been thinking about this and really don't know what would be the best approach. When I suggested generating this in CI/CD I was thinking the final config wasn't in the repo, but of course, it is so we should avoid writing it in CI/CD. On the other hand, we need to automate it somehow to prevent releasing a new version without updating the shareable configs properly.

I think what we can do is something similar to what to do to make sure exported rules are the expected ones etc: Add a test to `index.test.ts` which manually builds each shareable config from rules and compares them against the current shareable config generated in the codebase. This way, we avoid writing this on CI/CD but we prevent forgetting about updating those presets. What do you think? /cc @MichaelDeBoey